compiler/rustc_resolve/src/lib.rs

```diff
@@ -202,7 +204,12 @@ enum ResolutionError<'a> {
     /// Error E0431: `self` import can only appear in an import list with a non-empty prefix.
     SelfImportOnlyInImportListWithNonEmptyPrefix,
     /// Error E0433: failed to resolve.
-    FailedToResolve { label: String, suggestion: Option<Suggestion> },
+    FailedToResolve {
+        last_segment: Option<Segment>,
```

Comment:
```suggestion
        last_segment: Option<Symbol>,
```
Only the name seems to be used.